config/locales/fr.yml

```diff
@@ -249,7 +249,7 @@ fr:
       - vendredi
       - samedi
     formats:
-      default: "%d/%m/%Y"
+      default: "%d %B %Y"
```

Comment:
👌 This might change formatting in unintended places, but this shouldn’t break anything badly. If needed, we can always add more custom formats here.